pyjudge/visualize/__init__.py

```diff
@@ -132,15 +108,17 @@ def make_line_count(i, ps_arr=None):
 </div>
 """
 
+
 def create(json_data):
     """ Create visualized HTML with JSON input in dict(). """
     data = html_template_data_main
     if type(data) == bytes:
         data = data.decode('utf-8', 'ignore')
     data = mako.template.Template(
-        text = data,
-        input_encoding = 'utf-8').render(
-            json_data = json_data
-        )
-    data = vis_html_data.html_template_data_begin + data + vis_html_data.html_template_data_end
+        text=data,
+        input_encoding='utf-8').render(
+            json_data=json_data
+    )
+    data = vis_html_data.html_template_data_begin + \
+        data + vis_html_data.html_template_data_end
```

Comment:
The PEP8 way of wrapping expressions:

```suggestion
        text=data,
        input_encoding='utf-8').render(
            json_data=json_data
    )
    data = (vis_html_data.html_template_data_begin +
            data +
            vis_html_data.html_template_data_end)
```